river/Seat.zig

```diff
@@ -436,6 +446,11 @@ fn addPointer(self: Self, device: *wlr.InputDevice) void {
     self.cursor.wlr_cursor.attachInputDevice(device);
 }
 
+fn addTouch(self: *Self, device: *wlr.InputDevice) void {
+    self.touch.wlr_cursor.attachInputDevice(device);
```

Comment:
I quickly checked Sway again and I think they do not have a separate cursor for touch. What are the advantages of having a separate cursor instead of just using the one that is already there?

Reply:
And, since touch operations seem to be cursor operations in wlroots, why not have them in `Cursor.zig`?

Reply:
I was using the mouse cursor at first but changed that when you mentioned that touch is not supposed to move the cursor. However, since wlroots needs a cursor for the events, I needed to add a cursor anyway. So perhaps it makes sense to use the already existing mouse cursor since we only really need it for events.